### ISSUE TITLE
[Docs] Storybook theme is inaccessible

### ISSUE THREAD

**Issue by damienrobson-sage** [Sage/carbon#8011]
The default green used in the new Storybook theme fails colour contrast (187:1).

<img width="1004" height="334" alt="Image" src="https://github.com/user-attachments/assets/7a872487-0227-409f-9a53-7bd9f6987aa5" />
